src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionCommentFilterTest.java

```diff
@@ -451,7 +452,7 @@ public void testSuppressByIdAndMessage() throws Exception {
             createModuleConfig(SuppressionCommentFilter.class);
         filterConfig.addAttribute("offCommentFormat", "CSOFF (\\w+) \\(allow (\\w+)\\)");
         filterConfig.addAttribute("onCommentFormat", "CSON (\\w+)");
-        filterConfig.addAttribute("checkFormat", "$1");
+        filterConfig.addAttribute("idFormat", "$1");
```

Comment:
please do not change existing tests configuration, as you create new property please create new test methods.
please create also test for checkFormat and idFormat are defined together.

Reply:
I pushed a new commit where
* `testSuppressByIdAndMessage` uses `idFormat`
* `testSuppressByCheckAndMessage` uses `checkFormat`